youtube_dl/extractor/prosiebensat1.py

```diff
@@ -374,8 +366,7 @@ def _extract_clip(self, url, webpage):
             self._CLIPID_REGEXES, webpage, 'clip id')
         title = self._html_search_regex(self._TITLE_REGEXES, webpage, 'title')
         info = self._extract_video_info(url, clip_id)
-        description = self._html_search_regex(
-            self._DESCRIPTION_REGEXES, webpage, 'description', fatal=False)
+        description = self._og_search_description(webpage, default=None)
```

Comment:
Okay, So leaving the old one, and using the new one as a fallback when the old one hasn't succeeded?